locales/nl/validation.php

```diff
@@ -107,7 +107,7 @@
     'prohibits'            => 'Het veld :attribute verbiedt de aanwezigheid van :other.',
     'regex'                => ':Attribute formaat is ongeldig.',
     'required'             => ':Attribute is verplicht.',
-    'required_array_keys'  => 'The :attribute field must contain entries for: :values.',
+    'required_array_keys'  => 'Het :attribute veld moet waardes bevatten voor :values.',
```

Comment:
'Het' and 'veld' should be omitted, because it will never result in a grammatically correct sentence.

Incorrect: "Het e-mailadres veld (...)"
Correct: "Het e-mailadresveld (...)"

```suggestion
    'required_array_keys'  => ':Attribute moet waardes bevatten voor :values.',
```

This will be consistent with most of the other translations (I'll send a PR to remove "Het" + "veld" from the other translations).